### PR TITLE
enhanced PodsResourceTest

### PR DESCRIPTION
**Pull request by unterstein**
added PodsResource test for unauthorized requests

### CODE REVIEW

**Comment by jdef** [comment]
the comments indicate that task-killer will perform authorization. is there a need for double-authz?

**Comment by unterstein** [comment]
Right ... TaskKiller do. Will change this

**Comment by aquamatthias** [comment]
Thanks.
You test for authorized=false. Can you also test for authenticated=false?

**Comment by unterstein** [comment]
added authenticated false as well

File: src/test/scala/mesosphere/marathon/api/v2/PodsResourceTest.scala [hunk 1]
```diff
@@ -280,6 +279,116 @@ class PodsResourceTest extends AkkaUnitTest with Mockito {
 
       }
     }
+
+    "access without authentication is denied" when {
+      "An unauthenticated request" when {
```
Comment (aquamatthias) [inline src/test/scala/mesosphere/marathon/api/v2/PodsResourceTest.scala]: Nit: can you create 2 subsuites: one for unauthenticated one for unauthorized?

**Comment by aquamatthias** [comment]
💯 Thanks

**Pull request merged**